src/scroll-timeline-base.js

```diff
@@ -58,6 +58,32 @@ function calculateTimeRange(scrollTimeline) {
   return timeRange;
 }
 
+/**
+ * Calculates a scroll offset that corrects for writing modes, text direction
+ * and a logical orientation.
+ * @param scrollTimeline {ScrollTimeline}
+ * @param orientation {String}
+ * @returns {Number}
+ */
+function directionAwareScrollOffset(scrollSource, orientation) {
+  const style = getComputedStyle(scrollSource);
+  // All writing modes are vertical except for horizontal-tb.
+  // http://drafts.csswg.org/css-writing-modes-4/#block-flow
+  const horizontalWritingMode = style.writingMode == 'horizontal-tb';
+  let currentScrollOffset  = scrollSource.scrollTop;
```

Comment:
Looks like sieways-lr switches the inline direction to be from bottom to top, which may require flipping or taking the absolute value of the scrollTop. See https://developer.mozilla.org/en-US/docs/Web/CSS/writing-mode#Examples. We can add a TODO for this - it also seems to be unsupported in chrome.

Reply:
Added a TODO.